tools/codegen/api/types.py

```diff
@@ -483,14 +485,15 @@ def dispatcher_exprs(self) -> List[Expr]:
 
 # Helper functions
 
-def kernel_signature(f: NativeFunction, backend_index: BackendIndex) -> Union['NativeSignature', 'DispatcherSignature']:
+def kernel_signature(
```

Comment:
ditto re comment for `prefix` - also I'm not far enough into the PR yet to understand why it's a freestanding param and not part of the data model.